### PR TITLE
Added models and dbContext

### PR DESCRIPTION
**Pull request by DTA-Wijesekara**

### CODE REVIEW

File: AssetManagementSystem/Context/assetmanegementDbContext.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
+﻿using AssetManagementSystem.Models;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Identity.Client;
+
+namespace AssetManagementSystem.Context
+{
+    public class assetmanegementDbContext : DbContext
+    {
+        public assetmanegementDbContext(DbContextOptions dbContextOptions): base(dbContextOptions)
```
Comment (HasangaLakdinu) [inline AssetManagementSystem/Context/assetmanegementDbContext.cs:9]: assetManagementDbContext

Comment (DTA-Wijesekara) [reply AssetManagementSystem/Context/assetmanegementDbContext.cs]: resolved

File: AssetManagementSystem/appsettings.json [hunk 2]
```diff
@@ -5,5 +5,8 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
+  "ConnectionStrings": {
+    "dbstring": "Server=DASUN-THEEKSHAN;Database=AssetManegementOfficial;Trusted_Connection=True;TrustserverCertificate=True"
```
Comment (HasangaLakdinu) [inline AssetManagementSystem/appsettings.json:10]: for now, let's make this string a empty when we commit.. i will find a better solution later

Comment (DTA-Wijesekara) [reply AssetManagementSystem/appsettings.json]: I made the server name empty.
is it okay or do I need to empty the string fully?

Comment (HasangaLakdinu) [reply AssetManagementSystem/appsettings.json]: okay lets keep this as it is.

**Pull request merged**